### PR TITLE
[util-linux] switch to the util-linux organization

### PR DESCRIPTION
**Pull request by evverx**
The repository was moved recently

### CODE REVIEW

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: projects/util-linux/project.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@ sanitizers:
   - address
   - undefined
   - memory
-main_repo: 'https://github.com/karelzak/util-linux'
+main_repo: 'https://github.com/util-linux/util-linux'
```
Comment (evverx) [inline projects/util-linux/project.yaml:17]: I'm not sure why https://github.com/ossf/scorecard keeps saying that `util-linux` isn't fuzzed though. I wonder what else I should add here to let `scorecard` know that it's really fuzzed.

Comment (evverx) [reply projects/util-linux/project.yaml:17]: What's weird is that according to `scorecard` `selinux` isn't fuzzed either. I'm puzzled

Comment (evverx) [reply projects/util-linux/project.yaml:17]: I've just opened https://github.com/ossf/scorecard/issues/1231

**Pull request merged**